acceptance-tests/build.gradle.kts

```diff
@@ -5,7 +5,6 @@ import java.net.URL
 
 plugins {
     java
-    id("buildlogic.chromedriver")
```

Comment:
As Selenium does this I think this plugin can be deleted too.,

Reply:
Yes it's deleted there https://github.com/jenkinsci/gradle-plugin/pull/369/files#diff-00d1422a52d1cd37c7e2205657f1533c8db46efe0adcac04dc60cd157a6addda